### ISSUE TITLE
Drop rate super low

### ISSUE THREAD

**Issue by AZimmermann89** [Heishe/idleraiders-second-run#812]
I noticed that my drop rate was terribly low. I didnt get an artefact item in days.
Switching the Ancient Wisdom trait off and on didnt change it, so I exported and imported the game state.
I am synching my game state usually between computers.
Since then I already got an artefact item, so that "probably" fixed it.

**Comment by AZimmermann89** [comment]
Nope, still an issue. I left my rangers farm all weekend and I didn't even get a rare item, let alone an artefact. And that is with 10/10 increased drop chance activated on Tier 9.

**Comment by hamburml** [comment]
Which raids did you farm?

**Comment by AZimmermann89** [comment]
I let my raiders farm on map 100 and set it on 'automatic raiding'. I got drops from the raids, but not on the farm map.

**Comment by kbsoftwarellc** [comment]
What's the highest farm level you've reached since you last soft reset?

**Comment by AZimmermann89** [comment]
lvl 113, so I am not farming with mitigated drop chance.

**Comment by kbsoftwarellc** [comment]
do you know your average number of kills per day?

**Comment by AZimmermann89** [comment]
No, but I have the same issue if I farm on lvl 7 map only, to have maximum kills. I doubt it is based on that I do not kill enough. I had one artefact drop in the last two weeks as far as I recall.

**Comment by AZimmermann89** [comment]
Since the one artefact I got 13 days ago I didnt get a single one.
As I can not progress with farming items anymore, the only way is via WIAR, which is not a fun grind, so at this point I can only stop playing really.
